src/eko/kernels/evolution_integrals_qed.py

```diff
@@ -0,0 +1,303 @@
+r"""Compute evolution integrals needed for QED."""
+import numba as nb
+import numpy as np
+
+from .. import beta
+from . import evolution_integrals as ei
+
+
+@nb.njit(cache=True)
+def j00_qed(a1, a0, aem, nf):
+    r"""LO-LO QED exact evolution integral.
+
+    .. math::
+        j^{(0,0)}(a_s,a_s^0,aem) = \int\limits_{a_s^0}^{a_s} \frac{da_s'}{(\beta_0 + aem \beta_{0,1}) a_s'}
+           = \frac{\ln(a_s/a_s^0)}{\beta_0 + aem \beta_{0,1}}
+
+    Parameters
+    ----------
+    a1 : float
+        target coupling value
+    a0 : float
+        initial coupling value
+    aem : float
+        electromagnetic coupling value
+    nf : int
+        number of active flavors
+
+    Returns
+    -------
+    j00 : float
+        integral
+    """
+    beta0 = beta.beta_qcd((2, 0), nf) + aem * beta.beta_qcd((2, 1), nf)
+    return np.log(a1 / a0) / beta0
+
+
+@nb.njit(cache=True)
+def jm10(a1, a0, aem, nf):
+    r"""LO-LO QED exact evolution integral.
```

Comment:
please find a different name for the "m1" ... no idea which one though ... a stupid one would be "(1/N)LO"

Reply:
One idea that I had was to call it 0 and to shift all of the others by one unit

Reply:
On the other hand using the `-1` is the only way in which `jm1n` satisfies the definition of `j(m,n)` given at the top of the module

Reply:
> One idea that I had was to call it 0 and to shift all of the others by one unit

maybe this is the way to go: we label them by $j^{u,v} = \int \alpha_s^u/\beta^{v-2}_{QCD}$ - so first index shifts by 1 and second by two (since beta_QCD start at this order) ... also I was more worried about the docstring ... and if we follow this suggestion, then maybe we just drop the mention of "LO" (there is no unique LO anymore) and write something like ```[insert details here] evolution integral :math:`\sim \alpha^0/\beta^{(0)}` ```